src/hatti/views/table.cljs

```diff
@@ -82,21 +83,47 @@
   (let [clj-value (js->clj value :keywordize-keys true)]
     (forms/format-answer field clj-value language true)))
 
+(defn button-formatter
+  [row cell value columnDef dataContext]
+  (let [{:keys [owner project formid]} (:dataset-info @shared/app-state)
+        form-owner (last-url-param owner)
+        project-id (last-url-param project)
+        edit-link (str "/"
```

Comment:
the `url` defined in hatti doesn't https://github.com/onaio/hatti/blob/master/src/hatti/utils.cljs#L7 , adding it breaks other stuff that uses it.

Will modify what uses it to factor in the already prefixed "/" , similar to the one we use in zebra https://github.com/onaio/milia/blob/master/src/milia/utils/url.cljc#L13